air/src/options.rs

```diff
@@ -291,6 +311,10 @@ impl Deserializable for ProofOptions {
             FieldExtension::read_from(source)?,
             source.read_u8()? as usize,
             source.read_u8()? as usize,
+            match source.read_u8()? {
+                0 => BatchingType::Linear,
+                _ => BatchingType::Algebraic,
```

Comment:
We shouldn't need to use the `match` statement here - i.e., the following should work: `BatchingType::read_from(source)?`.

Reply:
same as above